static/format-sdb.sh

```diff
@@ -93,43 +104,12 @@ then
     sed -i 10q /etc/fstab > /dev/null
 fi
 
-# Mount it in fstab
-UUID=$(blkid /dev/sdb1 | awk '{ print $3 }')
-FSTAB="$UUID     $MOUNT_     btrfs   defaults 0       2"
-echo "# ncdata mount" >> /etc/fstab
-echo "$FSTAB" >> /etc/fstab
-check_command mount -a
-
```

Comment:
It seems to me that it doesn't stay persistent across reboots if you don't put it in fstab as well.

Run the script, reboot, and check `df -h`.

Reply:
Sorry, my bad, missed the last line. Please ignore this.

Reply:
Ok, I was pretty sure that was working. Nevermind

Reply:
tested again on a new machine. persistent mount is working.
![selezione_999 887](https://user-images.githubusercontent.com/7568057/39672460-c340adc0-512a-11e8-9dab-bb49c6795353.png)